cpp/src/distance/detail/pairwise_matrix/dispatch_canberra_double_double_double_int64_t.cu

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021-2023, NVIDIA CORPORATION.
```

Comment:
New files should only have the current year

Reply:
I've updated that script that generated these files to set the copyright year to 2024 only -